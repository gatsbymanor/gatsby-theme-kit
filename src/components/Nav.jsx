import React from 'react';

export default function Nav(props) {
  return (
    <nav id="nav">
      <ul>
        <li><a href="#intro" className="active">Introduction</a></li>
        <li><a href="#first">First Section</a></li>
        <li><a href="#second">Second Section</a></li>
        <li><a href="#cta">Get Started</a></li>
      </ul>
    </nav>
  )
}