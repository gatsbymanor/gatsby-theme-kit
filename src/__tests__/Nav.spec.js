import React from 'react';
import renderer from 'react-test-renderer';
import Nav from '../components/Nav';

describe('<Nav />', () => {
  const component = renderer.create(<Nav />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});