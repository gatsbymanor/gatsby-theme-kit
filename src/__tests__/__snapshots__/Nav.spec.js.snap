// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> should render correctly 1`] = `
<nav
  id="nav"
>
  <ul>
    <li>
      <a
        className="active"
        href="#intro"
      >
        Introduction
      </a>
    </li>
    <li>
      <a
        href="#first"
      >
        First Section
      </a>
    </li>
    <li>
      <a
        href="#second"
      >
        Second Section
      </a>
    </li>
    <li>
      <a
        href="#cta"
      >
        Get Started
      </a>
    </li>
  </ul>
</nav>
`;
